doc/polycli_ulxly_deposit-new.md

```diff
@@ -0,0 +1,117 @@
+# `polycli ulxly deposit-new`
+
+> Auto-generated documentation.
+
+## Table of Contents
+
+- [Description](#description)
+- [Usage](#usage)
+- [Flags](#flags)
+- [See Also](#see-also)
+
+## Description
+
+Make a uLxLy deposit transaction
+
+```bash
+polycli ulxly deposit-new [flags]
+```
+
+## Usage
+
+This command will attempt to send a deposit transaction to the bridge contract.
+
+```solidity
+    /**
+     * @notice Deposit add a new leaf to the merkle tree
+     * note If this function is called with a reentrant token, it would be possible to `claimTokens` in the same call
+     * Reducing the supply of tokens on this contract, and actually locking tokens in the contract.
+     * Therefore we recommend to third parties bridges that if they do implement reentrant call of `beforeTransfer` of some reentrant tokens
+     * do not call any external address in that case
+     * note User/UI must be aware of the existing/available networks when choosing the destination network
+     * @param destinationNetwork Network destination
+     * @param destinationAddress Address destination
+     * @param amount Amount of tokens
+     * @param token Token address, 0 address is reserved for ether
+     * @param forceUpdateGlobalExitRoot Indicates if the new global exit root is updated or not
+     * @param permitData Raw data of the call `permit` of the token
+     */
+    function bridgeAsset(
+        uint32 destinationNetwork,
+        address destinationAddress,
+        uint256 amount,
+        address token,
+        bool forceUpdateGlobalExitRoot,
+        bytes calldata permitData
+    );
+
+```
```

Comment:
I guess we can also support `bridgeMessage` and `bridgeMessageWeth` methods by simply adding a few flags like `--weth` and `--metadata`.